hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSImageFormatProtobuf.java

```diff
@@ -653,7 +641,12 @@ public int getInodesPerSubSection() {
       return inodesPerSubSection;
     }
 
-    /**
+    public OutputStream getSectionOutputStream() {
+      return sectionOutputStream;
+    }
+
+
+      /**
```

Comment:
Please remove extra blank line and keep alignment.

Reply:
https://github.com/apache/hadoop/pull/6929/commits/6f32a1db79dfe4d6cc5713fd4a164ccb54eb749c